cookbooks/cdo-home-ubuntu/recipes/default.rb

```diff
@@ -7,11 +7,19 @@
   '.bashrc',
   '.inputrc',
   '.profile',
+  '.vimrc'
 ].each do |file|
   template "/home/#{node[:current_user]}/#{file}" do
     source "#{file[1..-1]}.erb"
     mode '0644'
     user node[:current_user]
     group node[:current_user]
   end
+
+  template "/root/#{file}" do
+    source "#{file[1..-1]}.erb"
+    mode '0644'
+    user 'root'
+    group 'root'
+  end
```

Comment:
Not sure it's a concern, but will adding all of these files to `root` home folder have any risk of unintended consequences, e.g., on startup or when running `sudo`? I don't think it will be a problem, just thinking out loud and pausing on this for a bit.

At least maybe worth manually trying to enter a `sudo su` root login shell to make sure it doesn't fail to execute the scripts at all or anything.

Reply:
:+1 Tested `sudo su` and there are no problems that I can see.